cvat/apps/engine/views.py

```diff
@@ -39,6 +39,7 @@
 
 import cvat.apps.dataset_manager as dm
 import cvat.apps.dataset_manager.views  # pylint: disable=unused-import
+from cvat.apps.dataset_manager.views import IMPORT_CACHE_FAILED_TTL, IMPORT_CACHE_SUCCESS_TTL
```

Comment:
Why these variables are defined in the dataset manager? They seem nothing to do with datasets, and they are also relevant to backups. All of this looks like engine configuration parameters, so I suggest to place them either in settings or in the engine config.